zero_bin/verifier/src/cli.rs

```diff
@@ -1,15 +1,23 @@
-use std::path::PathBuf;
+use std::string;
 
-use clap::{Parser, ValueHint};
+use clap::{Parser, Subcommand, ValueHint};
 use zero_bin_common::prover_state::cli::CliProverStateConfig;
 
 #[derive(Parser)]
 pub(crate) struct Cli {
+    #[command(subcommand)]
+    pub(crate) command: Option<Command>,
+
     /// The file containing the proof to verify
     #[arg(short, long, value_hint = ValueHint::FilePath)]
-    pub(crate) file_path: PathBuf,
+    pub(crate) file_path: Option<string::String>,
```

Comment:
Why this change?

Reply:
since file_path is an optional arg now

Reply:
I think we should have better way to solve this.
For example if you run binary and put any parameters (missing or not) and then `--help`, it will print command line help and exit. I think in this case (for worker and verifier) we should be able to do the same. Maybe it does not need to be subcommand, just optional top level parameter that you would check first and print version and exit if `--version` is anywhere in the parameters.
In this case, we could use it in the same on all `leader`, `woker`, `rpc` and `verifier` without disrupting current parameters.
@Nashtare @muursh What do you think?

Reply:
I'd agree with @atanmarko, calling `--version` should mimic how `--help` args behave, i.e. stopping execution and returning the desired info.